FakeItEasy.ruleset

```diff
@@ -5,4 +5,17 @@
     <Rule Id="CA1020" Action="None" />
     <Rule Id="CA1305" Action="None" />
   </Rules>
+  <Rules AnalyzerId="StyleCop.Analyzer" RuleNamespace="StyleCop.Analyzer">

```

Comment:
It was a mistake. It should be StyleCop.Analyzers. Actually, I'm surprised it works with the wrong id... I guess the AnalyzerId is just ignored.